index.js

```diff
@@ -362,6 +364,14 @@
         }
       });
 
+  //  keys :: StrMap a -> Array String
+  var keys = function(strMap) {
+    if (typeof Object.keys === 'function') return Object.keys(strMap);
```

Comment:
Yeah, that's what I tried but the problem is that Mocha runs the tests in parallel so when you mess with the `Object` prototype it affects all the other tests.